Tools/Scripts/webkitpy/api_tests/runner.py

```diff
@@ -112,21 +112,25 @@ def command_for_port(port, args):
         return args
 
     @staticmethod
-    def _shard_tests(tests):
+    def _shard_tests(tests, force_shard):
         shards = {}
         for test in tests:
             shard_prefix = '.'.join(test.split('.')[:-1])
             if shard_prefix not in shards:
                 shards[shard_prefix] = []
             shards[shard_prefix].append(test)
+        if len(shards) == 1 or force_shard:
```

Comment:
As you wrote in the commit message:

> also if only one test suite is specified it shards those tests individually

This feels like a kinda significant behavioural change that developers might hit locally (e.g., when trying to reproduce something failing on the bots), and might want a way to disable.

I think it probably makes sense to make that as a separate change?

That said, I'm not r-'ing this in case you/Jonathan/others feel this is fine.

Reply:
> This feels like a kinda significant behavioural change that developers might hit locally (e.g., when trying to reproduce something failing on the bots), and might want a way to disable.

Shards are regrouped if they're classified in mutually_exclusive_groups so functionally if the test can run in parallel it will but if not then it won't.

Reply:
Could we just do `if force_shard:` here? I think the only reason we might want this behavior `if len(shards)` is if we start running this logic and EWS and want to force any newly added test to shard with it's test group if it wasn't in the special "system" category.

Reply:
I can change it to `if force_shard:`

But previously you had said
>  if a suite can be shared, we should shard it if only that subsuite is specified

Removing `len(shards) ` would not allow this